oap-server/server-library/library-util/src/main/java/org/apache/skywalking/oap/server/library/util/CollectionUtils.java

```diff
@@ -26,15 +26,15 @@
 public class CollectionUtils {
 
     public static boolean isEmpty(Map map) {
-        return map == null || map.size() == 0;
+        return map == null || map.isEmpty();
```

Comment:
In most list, `isEmpty` == `size() == 0`. Could you give me some benchmark to indicate this will improve performance?

Reply:
Hi, looking at the implementations of String#isEmpty() and/or different List#isEmpty() implementations we see mostly "== 0" statements which from a top level perspective I generally consider to be more performant than smaller or greater comparisons. Furthermore, using isEmpty() keeps one on the safe side independent of the JIT and from my personal point of view additionally makes more clear what the purpose of the line of code is.
Of course we could create microbenchmark tests however I do trust others when the statements are so clear: https://stackoverflow.com/questions/1508975/why-is-list-size0-slower-than-list-isempty-in-java.

Reply:
For making codes more clear, I would say YES for this PR. But I need to point out, even from your post(stackoverflow), there is not clear about `isEmpty` has better performance.
In list(we only use ArrayList and LinkedList), there is no performance difference. Also, in `HashMap`/`HashSet`, it is the same too. Even more, when `isEmpty` call `size() == 0`, this change causes a more level in method stack. I don't know whether this could trigger side effect, but at least, it may not be better.

So, right now, I would say this PR is only code style related, rather than performance improvement.

If you have more clear test results, which could show I am wrong, please point out. Or do I miss anything in your StackOverflow post?